Makefile

```diff
@@ -12,14 +12,27 @@ IMAGE_TAG ?= latest
 OPERATOR_IMAGE ?= cluster-network-addons-operator
 REGISTRY_IMAGE ?= cluster-network-addons-registry
 
+fmt: goimports whitespace
+
+goimports:
+	goimports -w ./pkg ./cmd
+
+whitespace:
+	./hack/whitespace.sh --fix
+
+check: whitespace-check vet goimports-check generate-api-check test
+
+whitespace-check:
+	./hack/whitespace.sh
+
 vet:
 	go vet ./pkg/... ./cmd/...
 
-fmt:
-	go fmt ./pkg/... ./cmd/...
+goimports-check:
```

Comment:
Same as vet but for goimports
check-goimports: $(cmd_sources) $(pkg_sources)
goimports -d ./pkg ./cmd
touch $@

Reply:
Done